tests/helpers/snapshots.ts

```diff
@@ -48,7 +48,7 @@ export const restoreInstanceSnapshot = async (page: Page, snapshot: string) => {
     .getByRole("button", { name: "Restore" })
     .click();
 
-  await page.waitForSelector(`text=Snapshot ${snapshot} restored.`);
+  await page.waitForSelector(`text=Snapshot ${snapshot} restored`);
```

Comment:
Shouldn't we update this assertion to be `text=Snapshot ${snapshot} restored for instance ${instance}.`? Although we'd just need to pass the instance name into the helper here. Same comment for the other tests?